src/styles/stars.scss

```diff
@@ -0,0 +1,14 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background: url("../images/star.svg") center no-repeat;
+}
```

Comment:
use sass nesting here

Reply:
not fixed:
![image](https://github.com/mate-academy/layout_product-cards/assets/86923089/41ec6f66-70a8-4db7-ae0f-fad182bf5418)